18.1.1 clang compilation issue

```
**for chromium-ozone-wayland & qemux86-64 target:** getting compilation issue, sharing the log below for reference
   **NOTE: Executing Tasks
   ERROR: chromium-ozone-wayland-121.0.6167.184-r0 do_compile: ExecutionError('/buildarea/eng1/rpandi/wrkspc/chromium/buildchromium/build_poky_master/tmp/work/core2-64-poky-linux/chromium-ozone-wayland/121.0.6167.184/temp/run.do_compile.878508', 1, None, None)
   ERROR: Logfile of failure stored in: /buildarea/eng1/rpandi/wrkspc/chromium/buildchromium/build_poky_master/tmp/work/core2-64-poky-linux/chromium-ozone-wayland/121.0.6167.184/temp/log.do_compile.878508
   Log data follows:
   | DEBUG: Executing shell function do_compile
   | ninja: error: '../../../recipe-sysroot-native/usr/lib/clang/latest/lib/linux/libclang_rt.builtins-x86_64.a', needed by 'yocto_native/obj/third_party/protobuf/libprotoc_lib.a', missing and no known rule to make it
   | WARNING: /buildarea/eng1/rpandi/wrkspc/chromium/buildchromium/build_poky_master/tmp/work/core2-64-poky-linux/chromium-ozone-wayland/121.0.6167.184/temp/run.do_compile.878508:146 exit 1 from 'ninja -v -j 64 chrome chrome_sandbox chromedriver.unstripped'
   | WARNING: Backtrace (BB generated script):**
```

I got this issue when i add master branch to my manifest without revision and meta-clang automatically takes from latest master branch commit. When i set revision in manifest to eebe4ff2e539f3ffb01c5060cc4ca8b226ea8b52 commit (lksctp-tools: Reset SECURITY_STACK_PROTECTOR with clang) , last before bump to 18.1.1 version, all is good.

I'm getting a similar issue. Is there anyway to work around it for now? Before sync'ing my repo it worked fine.

```
| [18013/63163] clang++ -MMD -MF yocto_native/obj/third_party/flatbuffers/compiler_files/binary_annotator.o.d -DUSE_UDEV -DUSE_AURA=1 -DUSE_GLIB=1 -DUSE_OZONE=1 -DOFFICIAL_BUILD -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DNO_UNWIND_TABLES -D_GNU_SOURCE -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_NONE -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -DFLATBUFFERS_LOCALE_INDEPENDENT=0 -I../../third_party/flatbuffers/src/grpc -I../.. -Iyocto_native/gen -I../../third_party/flatbuffers/src/include -I../../third_party/flatbuffers/src/src -I../../third_party/flatbuffers/src -I../../third_party/flatbuffers/generated -fno-delete-null-pointer-checks -fno-ident -fno-strict-aliasing -fstack-protector -fno-unwind-tables -fno-asynchronous-unwind-tables -fPIC -pthread -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../../tools/clang/crashreports -mllvm -instcombine-lower-dbg-declare=0 -ffp-contract=off -mno-outline -ffile-compilation-dir=. -ftrivial-auto-var-init=pattern -O2 -fdata-sections -ffunction-sections -fno-unique-section-names -fno-math-errno -fno-omit-frame-pointer -g0 -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-c++11-narrowing -Wno-unused-but-set-variable -Wno-misleading-indentation -Wno-missing-field-initializers -Wno-unused-parameter -Wno-psabi -Wloop-analysis -Wno-unneeded-internal-declaration -Wenum-compare-conditional -Wno-unknown-warning-option -Wno-ignored-pragma-optimize -Wno-deprecated-builtins -Wno-bitfield-constant-conversion -Wno-deprecated-this-capture -Wno-invalid-offsetof -Wno-vla-extension -Wno-thread-safety-reference-return -Wno-constant-conversion -Wno-shorten-64-to-32 -std=c++20 -Wno-trigraphs -gsimple-template-names -fno-exceptions -fno-rtti -fvisibility-inlines-hidden -isystem/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native/usr/include -isysroot=/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native -stdlib=libc++ -isystem/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native/usr/include -isysroot=/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native -stdlib=libc++ -O2 -pipe -c ../../third_party/flatbuffers/src/src/binary_annotator.cpp -o yocto_native/obj/third_party/flatbuffers/compiler_files/binary_annotator.o
| [18014/63163] python3 ../../ui/webui/resources/tools/bundle_js.py --host new-tab-page --input gen/chrome/browser/resources/new_tab_page/tsc --out_folder gen/chrome/browser/resources/new_tab_page/bundled --depfile gen/chrome/browser/resources/new_tab_page/build_bundle.d --target_name build_bundle --exclude chrome://resources/mojo/mojo/public/js/bindings.js //resources/mojo/mojo/public/js/bindings.js chrome://resources/mojo/mojo/public/mojom/base/time.mojom-lite.js //resources/mojo/mojo/public/mojom/base/time.mojom-lite.js chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js //resources/polymer/v3_0/polymer/polymer_bundled.min.js chrome://resources/js/cr.js //resources/js/cr.js chrome://resources/js/load_time_data.js //resources/js/load_time_data.js chrome://resources/ash/common/load_time_data.m.js //resources/ash/common/load_time_data.m.js chrome://resources/mwc/lit/index.js //resources/mwc/lit/index.js chrome://resources/cr_components/color_change_listener/color_change_listener.mojom-webui.js chrome://resources/cr_components/customize_themes/customize_themes.mojom-webui.js chrome://resources/cr_components/page_image_service/page_image_service.mojom-webui.js chrome://resources/cr_components/most_visited/most_visited.mojom-webui.js chrome://resources/cr_components/omnibox/omnibox.mojom-webui.js chrome://resources/js/browser_command/browser_command.mojom-webui.js chrome://resources/js/metrics_reporter/metrics_reporter.mojom-webui.js chrome://resources/mojo/mojo/public/mojom/base/string16.mojom-webui.js chrome://resources/mojo/mojo/public/mojom/base/text_direction.mojom-webui.js chrome://resources/mojo/mojo/public/mojom/base/time.mojom-webui.js chrome://resources/mojo/skia/public/mojom/skcolor.mojom-webui.js chrome://resources/mojo/url/mojom/url.mojom-webui.js cart.mojom-webui.js discount.mojom-webui.js chrome_cart.mojom-webui.js drive.mojom-webui.js feed.mojom-webui.js history_cluster_types.mojom-webui.js history_clusters.mojom-webui.js history_clusters_v2.mojom-webui.js history_clusters_layout_type.mojom-webui.js history_types.mojom-webui.js new_tab_page.mojom-webui.js photos.mojom-webui.js recipes.mojom-webui.js tab_resumption.mojom-webui.js --external_paths chrome://resources/polymer/v3_0/\|../../third_party/polymer/v3_0/components-chromium/ //resources/polymer/v3_0/\|../../third_party/polymer/v3_0/components-chromium/ chrome://resources/\|gen/ui/webui/resources/tsc/ //resources/\|gen/ui/webui/resources/tsc/ --js_module_in_files new_tab_page.js lazy_load.js
| FAILED: gen/chrome/browser/resources/new_tab_page/bundled/new_tab_page.rollup.js gen/chrome/browser/resources/new_tab_page/bundled/new_tab_page.rollup.js.map gen/chrome/browser/resources/new_tab_page/bundled/lazy_load.rollup.js gen/chrome/browser/resources/new_tab_page/bundled/lazy_load.rollup.js.map gen/chrome/browser/resources/new_tab_page/bundled/shared.rollup.js gen/chrome/browser/resources/new_tab_page/bundled/shared.rollup.js.map gen/chrome/browser/resources/new_tab_page/bundled/build_bundle_requestlist.txt gen/chrome/browser/resources/new_tab_page/bundled/rollup.config.mjs
| python3 ../../ui/webui/resources/tools/bundle_js.py --host new-tab-page --input gen/chrome/browser/resources/new_tab_page/tsc --out_folder gen/chrome/browser/resources/new_tab_page/bundled --depfile gen/chrome/browser/resources/new_tab_page/build_bundle.d --target_name build_bundle --exclude chrome://resources/mojo/mojo/public/js/bindings.js //resources/mojo/mojo/public/js/bindings.js chrome://resources/mojo/mojo/public/mojom/base/time.mojom-lite.js //resources/mojo/mojo/public/mojom/base/time.mojom-lite.js chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js //resources/polymer/v3_0/polymer/polymer_bundled.min.js chrome://resources/js/cr.js //resources/js/cr.js chrome://resources/js/load_time_data.js //resources/js/load_time_data.js chrome://resources/ash/common/load_time_data.m.js //resources/ash/common/load_time_data.m.js chrome://resources/mwc/lit/index.js //resources/mwc/lit/index.js chrome://resources/cr_components/color_change_listener/color_change_listener.mojom-webui.js chrome://resources/cr_components/customize_themes/customize_themes.mojom-webui.js chrome://resources/cr_components/page_image_service/page_image_service.mojom-webui.js chrome://resources/cr_components/most_visited/most_visited.mojom-webui.js chrome://resources/cr_components/omnibox/omnibox.mojom-webui.js chrome://resources/js/browser_command/browser_command.mojom-webui.js chrome://resources/js/metrics_reporter/metrics_reporter.mojom-webui.js chrome://resources/mojo/mojo/public/mojom/base/string16.mojom-webui.js chrome://resources/mojo/mojo/public/mojom/base/text_direction.mojom-webui.js chrome://resources/mojo/mojo/public/mojom/base/time.mojom-webui.js chrome://resources/mojo/skia/public/mojom/skcolor.mojom-webui.js chrome://resources/mojo/url/mojom/url.mojom-webui.js cart.mojom-webui.js discount.mojom-webui.js chrome_cart.mojom-webui.js drive.mojom-webui.js feed.mojom-webui.js history_cluster_types.mojom-webui.js history_clusters.mojom-webui.js history_clusters_v2.mojom-webui.js history_clusters_layout_type.mojom-webui.js history_types.mojom-webui.js new_tab_page.mojom-webui.js photos.mojom-webui.js recipes.mojom-webui.js tab_resumption.mojom-webui.js --external_paths chrome://resources/polymer/v3_0/\|../../third_party/polymer/v3_0/components-chromium/ //resources/polymer/v3_0/\|../../third_party/polymer/v3_0/components-chromium/ chrome://resources/\|gen/ui/webui/resources/tsc/ //resources/\|gen/ui/webui/resources/tsc/ --js_module_in_files new_tab_page.js lazy_load.js
| Traceback (most recent call last):
|   File "/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/../../ui/webui/resources/tools/bundle_js.py", line 285, in <module>
|     main(sys.argv[1:])
|   File "/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/../../ui/webui/resources/tools/bundle_js.py", line 258, in main
|     optimize_output = _optimize(args.input, args)
|   File "/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/../../ui/webui/resources/tools/bundle_js.py", line 224, in _optimize
|     js_module_out_files = _bundle(out_path, in_path, manifest_out_path,
|   File "/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/../../ui/webui/resources/tools/bundle_js.py", line 168, in _bundle
|     node.RunNode([node_modules.PathToRollup()] + rollup_args + [
|   File "/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/third_party/node/node.py", line 34, in RunNode
|     raise RuntimeError('Command \'%s\' failed\n%s' % (' '.join(cmd), err))
| RuntimeError: Command '/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/third_party/node/linux/node-linux-x64/bin/node /home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/third_party/node/node_modules/rollup/dist/bin/rollup /home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/gen/chrome/browser/resources/new_tab_page/tsc/new_tab_page.js /home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/gen/chrome/browser/resources/new_tab_page/tsc/lazy_load.js --chunkFileNames shared.rollup.js --format esm --dir /home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/gen/chrome/browser/resources/new_tab_page/bundled --entryFileNames [name].rollup.js --sourcemap --sourcemapExcludeSources --config /home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/gen/chrome/browser/resources/new_tab_page/bundled/rollup.config.mjs' failed
| 
| /home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/gen/chrome/browser/resources/new_tab_page/tsc/new_tab_page.js, /home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/chromium-121.0.6167.184/out/Release/gen/chrome/browser/resources/new_tab_page/tsc/lazy_load.js → gen/chrome/browser/resources/new_tab_page/bundled...
| [!] Error: Could not load gen/ui/webui/resources/tsc/cr_components/history_clusters/history_clusters_shared_style.css.js (imported by gen/chrome/browser/resources/new_tab_page/tsc/modules/history_clusters/tile.js): ENOENT: no such file or directory, open 'gen/ui/webui/resources/tsc/cr_components/history_clusters/history_clusters_shared_style.css.js'
| Error: Could not load gen/ui/webui/resources/tsc/cr_components/history_clusters/history_clusters_shared_style.css.js (imported by gen/chrome/browser/resources/new_tab_page/tsc/modules/history_clusters/tile.js): ENOENT: no such file or directory, open 'gen/ui/webui/resources/tsc/cr_components/history_clusters/history_clusters_shared_style.css.js'
| 
| 
| 
| [18015/63163] python3 ../../tools/typescript/ts_library.py --root_gen_dir ../../../../.. --root_src_dir ../../../../../../../../ --root_dir gen/chrome/browser/resources/side_panel/customize_chrome/preprocessed --gen_dir gen/chrome/browser/resources/side_panel/customize_chrome --out_dir gen/chrome/browser/resources/side_panel/customize_chrome/tsc --output_suffix build_ts --in_files app.ts appearance.ts button_label.ts cards.ts categories.ts check_mark_wrapper.ts chrome_colors.ts hover_button.ts shortcuts.ts themes.ts theme_snapshot.ts wallpaper_search/combobox/customize_chrome_combobox.ts wallpaper_search/wallpaper_search.ts wallpaper_search/wallpaper_search_tile.ts chrome_cart_proxy.ts common.ts customize_chrome_api_proxy.ts wallpaper_search/wallpaper_search_proxy.ts window_proxy.ts app.html.ts appearance.html.ts button_label.html.ts cards.html.ts categories.html.ts check_mark_wrapper.html.ts chrome_colors.html.ts hover_button.html.ts shortcuts.html.ts themes.html.ts theme_snapshot.html.ts wallpaper_search/combobox/customize_chrome_combobox.html.ts wallpaper_search/wallpaper_search.html.ts wallpaper_search/wallpaper_search_tile.html.ts chrome_cart.mojom-webui.ts customize_chrome.mojom-webui.ts wallpaper_search.mojom-webui.ts --definitions ../../../../../../../../tools/typescript/definitions/metrics_private.d.ts --raw_deps //chrome/browser/resources/side_panel/shared:build_ts //third_party/polymer/v3_0:library //ui/webui/resources/cr_components/color_change_listener:build_ts //ui/webui/resources/cr_components/help_bubble:build_ts //ui/webui/resources/cr_components/managed_dialog:build_ts //ui/webui/resources/cr_components/theme_color_picker:build_ts //ui/webui/resources/cr_elements:build_ts //ui/webui/resources/js:build_ts //ui/webui/resources/mojo:build_ts --deps ../shared/tsconfig_build_ts.json ../../../../../third_party/polymer/v3_0/tsconfig_library.json ../../../../../ui/webui/resources/cr_components/color_change_listener/tsconfig_build_ts.json ../../../../../ui/webui/resources/cr_components/help_bubble/tsconfig_build_ts.json ../../../../../ui/webui/resources/cr_components/managed_dialog/tsconfig_build_ts.json ../../../../../ui/webui/resources/cr_components/theme_color_picker/tsconfig_build_ts.json ../../../../../ui/webui/resources/cr_elements/tsconfig_build_ts.json ../../../../../ui/webui/resources/js/tsconfig_build_ts.json ../../../../../ui/webui/resources/mojo/tsconfig_build_ts.json --tsconfig_base ../../../../../../../../tools/typescript/tsconfig_base_polymer.json --composite --path_mappings /tools/typescript/definitions/\*\|../../../../../../../../tools/typescript/definitions/\* chrome://customize-chrome-side-panel.top-chrome/shared/\*\|../shared/tsc/\*
| [18016/63163] python3 ../../tools/typescript/ts_library.py --root_gen_dir ../../../../.. --root_src_dir ../../../../../../../../ --root_dir gen/chrome/browser/resources/side_panel/reading_list/preprocessed --gen_dir gen/chrome/browser/resources/side_panel/reading_list --out_dir gen/chrome/browser/resources/side_panel/reading_list/tsc --output_suffix build_ts --in_files app.ts reading_list_item.ts reading_list_api_proxy.ts app.html.ts reading_list_item.html.ts icons.html.ts reading_list.mojom-webui.ts --definitions ../../../../../../../../tools/typescript/definitions/metrics_private.d.ts --raw_deps //third_party/polymer/v3_0:library //ui/webui/resources/cr_components/color_change_listener:build_ts //ui/webui/resources/cr_components/help_bubble:build_ts //ui/webui/resources/cr_elements:build_ts //ui/webui/resources/js:build_ts //ui/webui/resources/mojo:build_ts --deps ../../../../../third_party/polymer/v3_0/tsconfig_library.json ../../../../../ui/webui/resources/cr_components/color_change_listener/tsconfig_build_ts.json ../../../../../ui/webui/resources/cr_components/help_bubble/tsconfig_build_ts.json ../../../../../ui/webui/resources/cr_elements/tsconfig_build_ts.json ../../../../../ui/webui/resources/js/tsconfig_build_ts.json ../../../../../ui/webui/resources/mojo/tsconfig_build_ts.json --tsconfig_base ../../../../../../../../tools/typescript/tsconfig_base_polymer.json --composite --path_mappings /tools/typescript/definitions/\*\|../../../../../../../../tools/typescript/definitions/\*
| [18017/63163] clang++ -MMD -MF yocto_native/obj/third_party/flatbuffers/compiler_files/idl_gen_binary.o.d -DUSE_UDEV -DUSE_AURA=1 -DUSE_GLIB=1 -DUSE_OZONE=1 -DOFFICIAL_BUILD -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DNO_UNWIND_TABLES -D_GNU_SOURCE -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_NONE -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -DFLATBUFFERS_LOCALE_INDEPENDENT=0 -I../../third_party/flatbuffers/src/grpc -I../.. -Iyocto_native/gen -I../../third_party/flatbuffers/src/include -I../../third_party/flatbuffers/src/src -I../../third_party/flatbuffers/src -I../../third_party/flatbuffers/generated -fno-delete-null-pointer-checks -fno-ident -fno-strict-aliasing -fstack-protector -fno-unwind-tables -fno-asynchronous-unwind-tables -fPIC -pthread -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../../tools/clang/crashreports -mllvm -instcombine-lower-dbg-declare=0 -ffp-contract=off -mno-outline -ffile-compilation-dir=. -ftrivial-auto-var-init=pattern -O2 -fdata-sections -ffunction-sections -fno-unique-section-names -fno-math-errno -fno-omit-frame-pointer -g0 -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-c++11-narrowing -Wno-unused-but-set-variable -Wno-misleading-indentation -Wno-missing-field-initializers -Wno-unused-parameter -Wno-psabi -Wloop-analysis -Wno-unneeded-internal-declaration -Wenum-compare-conditional -Wno-unknown-warning-option -Wno-ignored-pragma-optimize -Wno-deprecated-builtins -Wno-bitfield-constant-conversion -Wno-deprecated-this-capture -Wno-invalid-offsetof -Wno-vla-extension -Wno-thread-safety-reference-return -Wno-constant-conversion -Wno-shorten-64-to-32 -std=c++20 -Wno-trigraphs -gsimple-template-names -fno-exceptions -fno-rtti -fvisibility-inlines-hidden -isystem/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native/usr/include -isysroot=/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native -stdlib=libc++ -isystem/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native/usr/include -isysroot=/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native -stdlib=libc++ -O2 -pipe -c ../../third_party/flatbuffers/src/src/idl_gen_binary.cpp -o yocto_native/obj/third_party/flatbuffers/compiler_files/idl_gen_binary.o
| [18018/63163] clang++ -MMD -MF yocto_native/obj/third_party/flatbuffers/compiler_files/flatc.o.d -DUSE_UDEV -DUSE_AURA=1 -DUSE_GLIB=1 -DUSE_OZONE=1 -DOFFICIAL_BUILD -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DNO_UNWIND_TABLES -D_GNU_SOURCE -D_LIBCPP_HARDENING_MODE=_LIBCPP_HARDENING_MODE_NONE -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -DFLATBUFFERS_LOCALE_INDEPENDENT=0 -I../../third_party/flatbuffers/src/grpc -I../.. -Iyocto_native/gen -I../../third_party/flatbuffers/src/include -I../../third_party/flatbuffers/src/src -I../../third_party/flatbuffers/src -I../../third_party/flatbuffers/generated -fno-delete-null-pointer-checks -fno-ident -fno-strict-aliasing -fstack-protector -fno-unwind-tables -fno-asynchronous-unwind-tables -fPIC -pthread -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../../tools/clang/crashreports -mllvm -instcombine-lower-dbg-declare=0 -ffp-contract=off -mno-outline -ffile-compilation-dir=. -ftrivial-auto-var-init=pattern -O2 -fdata-sections -ffunction-sections -fno-unique-section-names -fno-math-errno -fno-omit-frame-pointer -g0 -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wall -Wno-unused-variable -Wno-c++11-narrowing -Wno-unused-but-set-variable -Wno-misleading-indentation -Wno-missing-field-initializers -Wno-unused-parameter -Wno-psabi -Wloop-analysis -Wno-unneeded-internal-declaration -Wenum-compare-conditional -Wno-unknown-warning-option -Wno-ignored-pragma-optimize -Wno-deprecated-builtins -Wno-bitfield-constant-conversion -Wno-deprecated-this-capture -Wno-invalid-offsetof -Wno-vla-extension -Wno-thread-safety-reference-return -Wno-constant-conversion -Wno-shorten-64-to-32 -std=c++20 -Wno-trigraphs -gsimple-template-names -fno-exceptions -fno-rtti -fvisibility-inlines-hidden -isystem/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native/usr/include -isysroot=/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native -stdlib=libc++ -isystem/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native/usr/include -isysroot=/home/parallels/gewa-platform/build/tmp/work/cortexa9hf-neon-imx-gel-linux-gnueabi/chromium-ozone-wayland/121.0.6167.184-r0/recipe-sysroot-native -stdlib=libc++ -O2 -pipe -c ../../third_party/flatbuffers/src/src/flatc.cpp -o yocto_native/obj/third_party/flatbuffers/compiler_files/flatc.o
| [18019/63163] python3 ../../ui/webui/resources/tools/minify_js.py --in_folder gen/ui/webui/resources/tsc/cr_elements --out_folder gen/ui/webui/resources/cr_elements/minified --out_manifest gen/ui/webui/resources/cr_elements/build_min_js_manifest.json --in_files cr_tab_box/cr_tab_box.js cr_tree/cr_tree.js cr_tree/cr_tree_item.js cr_a11y_announcer/cr_a11y_announcer.js cr_action_menu/cr_action_menu.js cr_button/cr_button.js cr_checkbox/cr_checkbox.js cr_chip/cr_chip.js cr_dialog/cr_dialog.js cr_drawer/cr_drawer.js cr_expand_button/cr_expand_button.js cr_feedback_buttons/cr_feedback_buttons.js cr_fingerprint/cr_fingerprint_progress_arc.js cr_grid/cr_grid.js cr_icon_button/cr_icon_button.js cr_input/cr_input.js cr_link_row/cr_link_row.js cr_loading_gradient/cr_loading_gradient.js cr_lottie/cr_lottie.js cr_textarea/cr_textarea.js cr_profile_avatar_selector/cr_profile_avatar_selector_grid.js cr_profile_avatar_selector/cr_profile_avatar_selector.js cr_radio_button/cr_card_radio_button.js cr_radio_button/cr_radio_button.js cr_radio_group/cr_radio_group.js cr_search_field/cr_search_field.js cr_segmented_button/cr_segmented_button.js cr_segmented_button/cr_segmented_button_option.js cr_slider/cr_slider.js cr_tabs/cr_tabs.js cr_toast/cr_toast_manager.js cr_toast/cr_toast.js cr_toggle/cr_toggle.js cr_toolbar/cr_toolbar.js cr_toolbar/cr_toolbar_search_field.js cr_toolbar/cr_toolbar_selection_overlay.js cr_url_list_item/cr_url_list_item.js cr_view_manager/cr_view_manager.js policy/cr_policy_indicator.js policy/cr_policy_pref_indicator.js policy/cr_tooltip_icon.js cr_splitter/cr_splitter.js cr_tree/cr_tree_base.js cr_auto_img/cr_auto_img.js cr_container_shadow_mixin.js cr_lazy_render/cr_lazy_render.js cr_menu_selector/cr_menu_selector.js cr_radio_button/cr_radio_button_mixin.js cr_scrollable_mixin.js cr_search_field/cr_search_field_mixin.js find_shortcut_mixin.js focus_row_mixin.js i18n_mixin.js list_property_update_mixin.js mouse_hoverable_mixin.js policy/cr_policy_indicator_mixin.js store_client/store_client.js web_ui_listener_mixin.js cr_tab_box/cr_tab_box.html.js cr_tree/cr_tree.html.js cr_tree/cr_tree_item.html.js cr_a11y_announcer/cr_a11y_announcer.html.js cr_action_menu/cr_action_menu.html.js cr_button/cr_button.html.js cr_checkbox/cr_checkbox.html.js cr_chip/cr_chip.html.js cr_dialog/cr_dialog.html.js cr_drawer/cr_drawer.html.js cr_expand_button/cr_expand_button.html.js cr_feedback_buttons/cr_feedback_buttons.html.js cr_fingerprint/cr_fingerprint_progress_arc.html.js cr_grid/cr_grid.html.js cr_icon_button/cr_icon_button.html.js cr_input/cr_input.html.js cr_link_row/cr_link_row.html.js cr_loading_gradient/cr_loading_gradient.html.js cr_lottie/cr_lottie.html.js cr_textarea/cr_textarea.html.js cr_profile_avatar_selector/cr_profile_avatar_selector_grid.html.js cr_profile_avatar_selector/cr_profile_avatar_selector.html.js cr_radio_button/cr_card_radio_button.html.js cr_radio_button/cr_radio_button.html.js cr_radio_group/cr_radio_group.html.js cr_search_field/cr_search_field.html.js cr_segmented_button/cr_segmented_button.html.js cr_segmented_button/cr_segmented_button_option.html.js cr_slider/cr_slider.html.js cr_tabs/cr_tabs.html.js cr_toast/cr_toast_manager.html.js cr_toast/cr_toast.html.js cr_toggle/cr_toggle.html.js cr_toolbar/cr_toolbar.html.js cr_toolbar/cr_toolbar_search_field.html.js cr_toolbar/cr_toolbar_selection_overlay.html.js cr_url_list_item/cr_url_list_item.html.js cr_view_manager/cr_view_manager.html.js policy/cr_policy_indicator.html.js policy/cr_policy_pref_indicator.html.js policy/cr_tooltip_icon.html.js cr_fingerprint/cr_fingerprint_icons.html.js icons.html.js mwb_shared_icons.html.js action_link.css.js cr_actionable_row_style.css.js cr_hidden_style.css.js cr_icons.css.js cr_nav_menu_item_style.css.js cr_page_host_style.css.js cr_radio_button/cr_radio_button_style.css.js cr_screen_reader_only.css.js cr_shared_style.css.js cr_shared_vars.css.js md_select.css.js mwb_element_shared_style.css.js mwb_shared_style.css.js mwb_shared_vars.css.js search_highlight_style.css.js cr_input/cr_input_style.css.js
| [18020/63163] python3 ../../tools/typescript/ts_library.py --root_gen_dir ../../../.. --root_src_dir ../../../../../../../ --root_dir gen/chrome/browser/resources/settings/preprocessed --gen_dir gen/chrome/browser/resources/settings --out_dir gen/chrome/browser/resources/settings/tsc --output_suffix build_ts --in_files a11y_page/a11y_page.ts about_page/about_page.ts ai_page/ai_page.ts appearance_page/appearance_fonts_page.ts appearance_page/appearance_page.ts appearance_page/home_url_input.ts autofill_page/address_edit_dialog.ts autofill_page/address_remove_confirmation_dialog.ts autofill_page/autofill_page.ts autofill_page/autofill_section.ts autofill_page/credit_card_edit_dialog.ts autofill_page/credit_card_list_entry.ts autofill_page/iban_edit_dialog.ts autofill_page/iban_list_entry.ts autofill_page/payments_list.ts autofill_page/payments_section.ts autofill_page/virtual_card_unenroll_dialog.ts basic_page/basic_page.ts clear_browsing_data_dialog/clear_browsing_data_dialog.ts clear_browsing_data_dialog/history_deletion_dialog.ts clear_browsing_data_dialog/passwords_deletion_dialog.ts controls/settings_checkbox_list_entry.ts controls/settings_checkbox.ts downloads_page/downloads_page.ts on_startup_page/on_startup_page.ts on_startup_page/startup_url_dialog.ts on_startup_page/startup_url_entry.ts on_startup_page/startup_urls_page.ts performance_page/battery_page.ts performance_page/performance_page.ts performance_page/speed_page.ts performance_page/tab_discard_exception_add_dialog.ts performance_page/tab_discard_exception_add_input.ts performance_page/tab_discard_exception_current_sites_list.ts performance_page/tab_discard_exception_edit_dialog.ts performance_page/tab_discard_exception_edit_input.ts performance_page/tab_discard_exception_entry.ts performance_page/tab_discard_exception_list.ts performance_page/tab_discard_exception_tabbed_add_dialog.ts people_page/page_content_page.ts people_page/people_page.ts people_page/signout_dialog.ts people_page/sync_account_control.ts people_page/sync_controls.ts people_page/sync_encryption_options.ts people_page/sync_page.ts privacy_page/anti_abuse_page.ts privacy_page/collapse_radio_button.ts privacy_page/cookies_page.ts privacy_page/do_not_track_toggle.ts privacy_page/personalization_options.ts privacy_page/preloading_page.ts privacy_page/privacy_guide/privacy_guide_completion_fragment.ts privacy_page/privacy_guide/privacy_guide_completion_link_row.ts privacy_page/privacy_guide/privacy_guide_cookies_fragment.ts privacy_page/privacy_guide/privacy_guide_description_item.ts privacy_page/privacy_guide/privacy_guide_dialog.ts privacy_page/privacy_guide/privacy_guide_history_sync_fragment.ts privacy_page/privacy_guide/privacy_guide_msbb_fragment.ts privacy_page/privacy_guide/privacy_guide_page.ts privacy_page/privacy_guide/privacy_guide_promo.ts privacy_page/privacy_guide/privacy_guide_preload_fragment.ts privacy_page/privacy_guide/privacy_guide_safe_browsing_fragment.ts privacy_page/privacy_guide/privacy_guide_search_suggestions_fragment.ts privacy_page/privacy_guide/privacy_guide_welcome_fragment.ts privacy_page/privacy_guide/step_indicator.ts privacy_page/privacy_page.ts privacy_page/security_keys_bio_enroll_dialog.ts privacy_page/security_keys_credential_management_dialog.ts privacy_page/security_keys_pin_field.ts privacy_page/security_keys_reset_dialog.ts privacy_page/security_keys_set_pin_dialog.ts privacy_page/security_keys_subpage.ts privacy_page/security_keys_phones_subpage.ts privacy_page/security_keys_phones_list.ts privacy_page/security_keys_phones_dialog.ts privacy_page/security_page.ts privacy_sandbox/privacy_sandbox_ad_measurement_subpage.ts privacy_sandbox/privacy_sandbox_fledge_subpage.ts privacy_sandbox/privacy_sandbox_interest_item.ts privacy_sandbox/privacy_sandbox_page.ts privacy_sandbox/privacy_sandbox_topics_subpage.ts reset_page/reset_page.ts reset_page/reset_profile_banner.ts reset_page/reset_profile_dialog.ts safety_check_page/safety_check_child.ts safety_check_page/safety_check_extensions.ts safety_check_page/safety_check_extensions_child.ts safety_check_page/safety_check_notification_permissions.ts safety_check_page/safety_check_page.ts safety_check_page/safety_check_passwords_child.ts safety_check_page/safety_check_safe_browsing_child.ts safety_check_page/safety_check_unused_site_permissions.ts safety_check_page/safety_check_updates_child.ts safety_hub/safety_hub_card.ts safety_hub/safety_hub_entry_point.ts safety_hub/safety_hub_module.ts safety_hub/extensions_module.ts safety_hub/safety_hub_page.ts safety_hub/notification_permissions_module.ts safety_hub/unused_site_permissions_module.ts search_engines_page/omnibox_extension_entry.ts search_engines_page/search_engine_edit_dialog.ts search_engines_page/search_engine_entry.ts search_engines_page/search_engines_list.ts search_engines_page/search_engines_page.ts search_page/search_engine_list_dialog.ts search_page/search_page.ts settings_main/settings_main.ts settings_menu/settings_menu.ts settings_page/settings_animated_pages.ts settings_page/settings_section.ts settings_page/settings_subpage.ts settings_ui/settings_ui.ts simple_confirmation_dialog.ts site_favicon.ts site_settings/add_site_dialog.ts site_settings/all_sites.ts site_settings/category_setting_exceptions.ts site_settings/chooser_exception_list_entry.ts site_settings/chooser_exception_list.ts site_settings/site_details_permission_device_entry.ts site_settings/edit_exception_dialog.ts site_settings/file_system_site_details.ts site_settings/file_system_site_entry.ts site_settings/file_system_site_entry_item.ts site_settings/file_system_site_list.ts site_settings/media_picker.ts site_settings/review_notification_permissions.ts site_settings_page/recent_site_permissions.ts site_settings_page/site_settings_list.ts site_settings_page/site_settings_page.ts site_settings_page/unused_site_permissions.ts site_settings/pdf_documents.ts site_settings/protocol_handlers.ts site_settings/settings_category_default_radio_group.ts site_settings/site_data.ts site_settings/site_details_permission.ts site_settings/site_details.ts site_settings/site_entry.ts site_settings/site_list_entry.ts site_settings/site_list.ts site_settings/storage_access_static_site_list_entry.ts site_settings/storage_access_site_list_entry.ts site_settings/storage_access_site_list.ts site_settings/zoom_levels.ts people_page/manage_profile.ts relaunch_confirmation_dialog.ts languages_page/add_languages_dialog.ts languages_page/languages_page.ts languages_page/spell_check_page.ts languages_page/translate_page.ts system_page/system_page.ts a11y_page/live_caption_section.ts a11y_page/live_translate_section.ts default_browser_page/default_browser_page.ts people_page/import_data_dialog.ts languages_page/edit_dictionary_page.ts a11y_page/captions_subpage.ts a11y_page/a11y_browser_proxy.ts about_page/about_page_browser_proxy.ts appearance_page/appearance_browser_proxy.ts autofill_page/address_edit_dialog_components.ts autofill_page/autofill_manager_proxy.ts autofill_page/password_manager_proxy.ts autofill_page/payments_manager_proxy.ts base_mixin.ts clear_browsing_data_dialog/clear_browsing_data_browser_proxy.ts controls/settings_idle_load.ts downloads_page/downloads_browser_proxy.ts ensure_lazy_loaded.ts focus_config.ts global_scroll_target_mixin.ts hats_browser_proxy.ts i18n_setup.ts lazy_load.ts metrics_browser_proxy.ts on_startup_page/on_startup_browser_proxy.ts on_startup_page/startup_urls_page_browser_proxy.ts page_visibility.ts performance_page/constants.ts performance_page/discard_timer_options.ts performance_page/performance_browser_proxy.ts performance_page/performance_metrics_proxy.ts performance_page/tab_discard_exception_validation_mixin.ts privacy_page/privacy_guide/constants.ts privacy_page/privacy_guide/privacy_guide_availability_mixin.ts privacy_page/privacy_guide/privacy_guide_browser_proxy.ts privacy_sandbox/privacy_sandbox_browser_proxy.ts privacy_page/security_keys_browser_proxy.ts relaunch_mixin.ts reset_page/reset_browser_proxy.ts route.ts router.ts safety_check_page/safety_check_browser_proxy.ts safety_check_page/safety_check_extensions_browser_proxy.ts safety_hub/safety_hub_browser_proxy.ts search_engines_page/search_engines_browser_proxy.ts search_settings.ts settings.ts settings_page/main_page_mixin.ts site_settings/constants.ts site_settings/site_settings_mixin.ts site_settings/site_settings_prefs_browser_proxy.ts site_settings/website_usage_browser_proxy.ts site_settings_page/site_settings_page_util.ts tooltip_mixin.ts default_browser_page/default_browser_browser_proxy.ts people_page/import_data_browser_proxy.ts languages_page/languages.ts languages_page/languages_browser_proxy.ts languages_page/languages_settings_metrics_proxy.ts languages_page/languages_types.ts people_page/manage_profile_browser_proxy.ts system_page/system_page_browser_proxy.ts a11y_page/a11y_page.html.ts about_page/about_page.html.ts ai_page/ai_page.html.ts appearance_page/appearance_fonts_page.html.ts appearance_page/appearance_page.html.ts appearance_page/home_url_input.html.ts autofill_page/address_edit_dialog.html.ts autofill_page/address_remove_confirmation_dialog.html.ts autofill_page/autofill_page.html.ts autofill_page/autofill_section.html.ts autofill_page/credit_card_edit_dialog.html.ts autofill_page/credit_card_list_entry.html.ts autofill_page/iban_edit_dialog.html.ts autofill_page/iban_list_entry.html.ts autofill_page/payments_list.html.ts autofill_page/payments_section.html.ts autofill_page/virtual_card_unenroll_dialog.html.ts basic_page/basic_page.html.ts clear_browsing_data_dialog/clear_browsing_data_dialog.html.ts clear_browsing_data_dialog/history_deletion_dialog.html.ts clear_browsing_data_dialog/passwords_deletion_dialog.html.ts controls/settings_checkbox_list_entry.html.ts controls/settings_checkbox.html.ts downloads_page/downloads_page.html.ts on_startup_page/on_startup_page.html.ts on_startup_page/startup_url_dialog.html.ts on_startup_page/startup_url_entry.html.ts on_startup_page/startup_urls_page.html.ts performance_page/battery_page.html.ts performance_page/performance_page.html.ts performance_page/speed_page.html.ts performance_page/tab_discard_exception_add_dialog.html.ts performance_page/tab_discard_exception_add_input.html.ts performance_page/tab_discard_exception_current_sites_list.html.ts performance_page/tab_discard_exception_edit_dialog.html.ts performance_page/tab_discard_exception_edit_input.html.ts performance_page/tab_discard_exception_entry.html.ts performance_page/tab_discard_exception_list.html.ts performance_page/tab_discard_exception_tabbed_add_dialog.html.ts people_page/page_content_page.html.ts people_page/people_page.html.ts people_page/signout_dialog.html.ts people_page/sync_account_control.html.ts people_page/sync_controls.html.ts people_page/sync_encryption_options.html.ts people_page/sync_page.html.ts privacy_page/anti_abuse_page.html.ts privacy_page/collapse_radio_button.html.ts privacy_page/cookies_page.html.ts privacy_page/do_not_track_toggle.html.ts privacy_page/personalization_options.html.ts privacy_page/preloading_page.html.ts privacy_page/privacy_guide/privacy_guide_completion_fragment.html.ts privacy_page/privacy_guide/privacy_guide_completion_link_row.html.ts privacy_page/privacy_guide/privacy_guide_cookies_fragment.html.ts privacy_page/privacy_guide/privacy_guide_description_item.html.ts privacy_page/privacy_guide/privacy_guide_dialog.html.ts privacy_page/privacy_guide/privacy_guide_history_sync_fragment.html.ts privacy_page/privacy_guide/privacy_guide_msbb_fragment.html.ts privacy_page/privacy_guide/privacy_guide_page.html.ts privacy_page/privacy_guide/privacy_guide_promo.html.ts privacy_page/privacy_guide/privacy_guide_preload_fragment.html.ts privacy_page/privacy_guide/privacy_guide_safe_browsing_fragment.html.ts privacy_page/privacy_guide/privacy_guide_search_suggestions_fragment.html.ts privacy_page/privacy_guide/privacy_guide_welcome_fragment.html.ts privacy_page/privacy_guide/step_indicator.html.ts privacy_page/privacy_page.html.ts privacy_page/security_keys_bio_enroll_dialog.html.ts privacy_page/security_keys_credential_management_dialog.html.ts privacy_page/security_keys_pin_field.html.ts privacy_page/security_keys_reset_dialog.html.ts privacy_page/security_keys_set_pin_dialog.html.ts privacy_page/security_keys_subpage.html.ts privacy_page/security_keys_phones_subpage.html.ts privacy_page/security_keys_phones_list.html.ts privacy_page/security_keys_phones_dialog.html.ts privacy_page/security_page.html.ts privacy_sandbox/privacy_sandbox_ad_measurement_subpage.html.ts privacy_sandbox/privacy_sandbox_fledge_subpage.html.ts privacy_sandbox/privacy_sandbox_interest_item.html.ts privacy_sandbox/privacy_sandbox_page.html.ts privacy_sandbox/privacy_sandbox_topics_subpage.html.ts reset_page/reset_page.html.ts reset_page/reset_profile_banner.html.ts reset_page/reset_profile_dialog.html.ts safety_check_page/safety_check_child.html.ts safety_check_page/safety_check_extensions.html.ts safety_check_page/safety_check_extensions_child.html.ts safety_check_page/safety_check_notification_permissions.html.ts safety_check_page/safety_check_page.html.ts safety_check_page/safety_check_passwords_child.html.ts safety_check_page/safety_check_safe_browsing_child.html.ts safety_check_page/safety_check_unused_site_permissions.html.ts safety_check_page/safety_check_updates_child.html.ts safety_hub/safety_hub_card.html.ts safety_hub/safety_hub_entry_point.html.ts safety_hub/safety_hub_module.html.ts safety_hub/extensions_module.html.ts safety_hub/safety_hub_page.html.ts safety_hub/notification_permissions_module.html.ts safety_hub/unused_site_permissions_module.html.ts search_engines_page/omnibox_extension_entry.html.ts search_engines_page/search_engine_edit_dialog.html.ts search_engines_page/search_engine_entry.html.ts search_engines_page/search_engines_list.html.ts search_engines_page/search_engines_page.html.ts search_page/search_engine_list_dialog.html.ts search_page/search_page.html.ts settings_main/settings_main.html.ts settings_menu/settings_menu.html.ts settings_page/settings_animated_pages.html.ts settings_page/settings_section.html.ts settings_page/settings_subpage.html.ts settings_ui/settings_ui.html.ts simple_confirmation_dialog.html.ts site_favicon.html.ts site_settings/add_site_dialog.html.ts site_settings/all_sites.html.ts site_settings/category_setting_exceptions.html.ts site_settings/chooser_exception_list_entry.html.ts site_settings/chooser_exception_list.html.ts site_settings/site_details_permission_device_entry.html.ts site_settings/edit_exception_dialog.html.ts site_settings/file_system_site_details.html.ts site_settings/file_system_site_entry.html.ts site_settings/file_system_site_entry_item.html.ts site_settings/file_system_site_list.html.ts site_settings/media_picker.html.ts site_settings/review_notification_permissions.html.ts site_settings_page/recent_site_permissions.html.ts site_settings_page/site_settings_list.html.ts site_settings_page/site_settings_page.html.ts site_settings_page/unused_site_permissions.html.ts site_settings/pdf_documents.html.ts site_settings/protocol_handlers.html.ts site_settings/settings_category_default_radio_group.html.ts site_settings/site_data.html.ts site_settings/site_details_permission.html.ts site_settings/site_details.html.ts site_settings/site_entry.html.ts site_settings/site_list_entry.html.ts site_settings/site_list.html.ts site_settings/storage_access_static_site_list_entry.html.ts site_settings/storage_access_site_list_entry.html.ts site_settings/storage_access_site_list.html.ts site_settings/zoom_levels.html.ts people_page/manage_profile.html.ts relaunch_confirmation_dialog.html.ts languages_page/add_languages_dialog.html.ts languages_page/languages_page.html.ts languages_page/spell_check_page.html.ts languages_page/translate_page.html.ts system_page/system_page.html.ts a11y_page/live_caption_section.html.ts a11y_page/live_translate_section.html.ts default_browser_page/default_browser_page.html.ts people_page/import_data_dialog.html.ts languages_page/edit_dictionary_page.html.ts a11y_page/captions_subpage.html.ts icons.html.ts site_settings/all_sites_icons.html.ts settings_columned_section.css.ts settings_page_styles.css.ts settings_shared.css.ts settings_vars.css.ts autofill_page/passwords_shared.css.ts privacy_page/privacy_guide/privacy_guide_fragment_shared.css.ts search_engines_page/search_engine_entry.css.ts site_settings/clear_storage_dialog_shared.css.ts site_settings_page/site_review_shared.css.ts --definitions ../../../../../../../tools/typescript/definitions/autofill_private.d.ts ../../../../../../../tools/typescript/definitions/chrome_send.d.ts ../../../../../../../tools/typescript/definitions/language_settings_private.d.ts ../../../../../../../tools/typescript/definitions/management.d.ts ../../../../../../../tools/typescript/definitions/metrics_private.d.ts ../../../../../../../tools/typescript/definitions/passwords_private.d.ts ../../../../../../../tools/typescript/definitions/pending.d.ts ../../../../../../../tools/typescript/definitions/settings_private.d.ts --raw_deps //chrome/browser/resources/settings_shared:build_ts //third_party/polymer/v3_0:library //ui/webui/resources/cr_components/customize_color_scheme_mode:build_ts //ui/webui/resources/cr_components/help_bubble:build_ts //ui/webui/resources/cr_components/localized_link:build_ts //ui/webui/resources/cr_components/managed_dialog:build_ts //ui/webui/resources/cr_components/managed_footnote:build_ts //ui/webui/resources/cr_components/settings_prefs:build_ts //ui/webui/resources/cr_elements:build_ts //ui/webui/resources/js:build_ts //ui/webui/resources/cr_components/customize_themes:build_ts //ui/webui/resources/cr_components/theme_color_picker:build_ts //ui/webui/resources/cr_components/certificate_manager:build_ts --deps ../settings_shared/tsconfig_build_ts.json ../../../../third_party/polymer/v3_0/tsconfig_library.json ../../../../ui/webui/resources/cr_components/customize_color_scheme_mode/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_components/help_bubble/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_components/localized_link/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_components/managed_dialog/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_components/managed_footnote/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_components/settings_prefs/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_elements/tsconfig_build_ts.json ../../../../ui/webui/resources/js/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_components/customize_themes/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_components/theme_color_picker/tsconfig_build_ts.json ../../../../ui/webui/resources/cr_components/certificate_manager/tsconfig_build_ts.json --tsconfig_base ../../../../../../../tools/typescript/tsconfig_base_polymer.json --composite --path_mappings /tools/typescript/definitions/\*\|../../../../../../../tools/typescript/definitions/\* /shared/settings/\*\|../settings_shared/tsc/\*
| ninja: build stopped: subcommand failed.
| WARNING: exit code 1 from a shell command.
ERROR: Task (/home/parallels/gewa-platform/sources/meta-browser/meta-chromium/recipes-browser/chromium/chromium-ozone-wayland_121.0.6167.184.bb:do_compile) failed with exit code '1'
NOTE: Tasks Summary: Attempted 5036 tasks of which 5023 didn't need to be rerun and 1 failed.
NOTE: Writing buildhistory
NOTE: Writing buildhistory took: 2 seconds

Summary: 1 task failed:
  /home/parallels/gewa-platform/sources/meta-browser/meta-chromium/recipes-browser/chromium/chromium-ozone-wayland_121.0.6167.184.bb:do_compile
Summary: There were 2 WARNING messages.
Summary: There was 1 ERROR message, returning a non-zero exit code.
```

--- a/meta-chromium/recipes-browser/chromium/chromium-gn.inc
+++ b/meta-chromium/recipes-browser/chromium/chromium-gn.inc
@@ -289,8 +289,8 @@ GN_ARGS += ' \
 '
 
 # Make sure Chromium is able to find clang libraries. See
-# 0023-Use-the-correct-path-to-libclang_rt.builtins.a.patch and the
-# add_clang_symlink and copy_clang_library tasks for more context.
+# 0008-Use-the-correct-path-to-libclang_rt.builtins.a.patch and the
+# add_clang_latest and copy_clang_library tasks for more context.
 GN_ARGS += ' \
         clang_base_path="${@clang_install_path(d)}" \
         clang_version="latest" \
@@ -436,33 +436,36 @@ do_add_nodejs_symlink () {
 }
 addtask add_nodejs_symlink after do_configure before do_compile
 
-do_add_clang_symlink () {
-    # Creates a `latest` symlink in the native sysroot's /usr/lib/clang
-    # directory that points to /usr/lib/clang/$CLANG_VERSION. Chromium manually
-    # links against libclang_rt.builtins.a and uses the `clang_version` GN
-    # variable to find it. This allows us to set it to the same value for all
+do_add_clang_latest () {
+    # Creates a `latest` directory in the native sysroot's /usr/lib/clang
+    # directory that is a copy of /usr/lib/clang/$CLANG_VERSION. Chromium
+    # manually links against libclang_rt.builtins.a and uses the `clang_version`
+    # GN variable to find it. This allows us to set it to the same value for all
     # Yocto releases.
     cd "${STAGING_LIBDIR_NATIVE}/clang"
+    rm -rf latest
     # find the directory containing the library
     for dir in *; do
         if [ -n "$(find $dir -name 'libclang_rt.builtins*')" ] ; then
-            ln -sf "$dir" latest
+            cp -r "$dir" latest
             break
         fi
     done
 }
-addtask add_clang_symlink after do_configure before do_compile
+addtask add_clang_latest after do_configure before do_compile
 
 do_copy_clang_library () {
     # Chromium needs to link against libclang_rt.builtins.a for both host and
     # target code, and expects to find both libraries in the same directory
-    # (thanks to 0023-Use-the-correct-path-to-libclang_rt.builtins.a.patch).
-    cd "${STAGING_LIBDIR}"
-    lib_file="$(find clang -name 'libclang_rt.builtins*')"
-    lib_dir="$(dirname $lib_file)"
-    cp "$lib_file" "${STAGING_LIBDIR_NATIVE}/$lib_dir"
+    # (thanks to 0008-Use-the-correct-path-to-libclang_rt.builtins.a.patch).
+    cd "${STAGING_LIBDIR}/clang"
+    # lib_file = "./$CLANG_VERSION/lib/linux/libclang_rt.builtins-$ARCH.a"
+    lib_file="$(find . -name 'libclang_rt.builtins*')"
+    # stripped_lib_file = "lib/linux/libclang_rt.builtins-$ARCH.a"
+    stripped_lib_file="${lib_file#*/*/}"
+    cp "$lib_file" "${STAGING_LIBDIR_NATIVE}/clang/latest/${stripped_lib_file}"
 }
-addtask copy_clang_library after do_configure before do_compile
+addtask copy_clang_library after do_add_clang_latest before do_compile
 
 do_copy_target_rustlibs () {
     # Chromium needs a single Rust sysroot that contains the rustlibs for both

